### PR TITLE
IDVA5-838 Documents used to check identity IDV

### PR DESCRIPTION
**Pull request by ttingle-ch**
- Added screen with document checkboxes
- Added service to handle single checkbox and add it to an array
- Added unit tests

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/controllers/identityDocumentsCheckedGroup1Controller.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { formatValidationError, getPageProperties } from "../validations/validation";
+import { CheckedDocumentsService } from "../services/checkedDocumentsService";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const session: Session = req.session as any as Session;
+    const previousPage: string = addLangToUrl(BASE_URL + HOW_IDENTITY_DOCUMENTS_CHECKED, lang);
+    const currentUrl: string = BASE_URL + WHICH_IDENTITY_DOCS_CHECKED_GROUP1;
+    const clientData: ClientData = session.getExtraData(USER_DATA) ? session.getExtraData(USER_DATA)! : {};
```
[the remark this answers is not in the document]
Comment (ttingle-ch) [reply src/controllers/identityDocumentsCheckedGroup1Controller.ts]: I have updated this

File: src/services/checkedDocumentsService.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Session } from "@companieshouse/node-session-handler";
+import { ClientData } from "../model/ClientData";
+import { USER_DATA } from "../utils/constants";
+import { Request } from "express";
+
+export class CheckedDocumentsService {
+    public saveDocumentsToSession (req: Request, clientData: ClientData, documents: string | string[]): void {
```
[the remark this answers is not in the document]
Comment (ttingle-ch) [reply src/services/checkedDocumentsService.ts]: updated the method name

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sampankumar — APPROVED** [review]

**Review by atedros-ch — APPROVED** [review]
LGTM

**Pull request merged**